src/common/problem_definition.cc

```diff
@@ -121,6 +121,8 @@ void DeclareParameters (dealii::ParameterHandler &local_prm) {
     local_prm.declare_entry ("fuel pin material id file name", "pin_id.txt",
         dealii::Patterns::FileName(),
         "file name for pin material id map for pin-resolved calculations");
+    local_prm.declare_entry("material id file name map", "",
+                            dealii::Patterns::Map(dealii::Patterns::Integer(),dealii::Patterns::FileName(dealii::Patterns::FileName::input)));                            
```

Comment:
Break Line 125, it's too long

Reply:
You can also move `dealii::xxxx` towards left until the fourth indent space, read Google Style if you want to.

Reply:
I meant to fix that, thanks.